build.gradle

```diff
@@ -16,4 +20,82 @@ dependencies {
     compile 'org.apache.httpcomponents:httpclient:4.5.3'
     testCompile 'junit:junit:4.12'
     testCompile 'org.json:json:20180813'
+}
+
+bintray {
+    user = System.getenv('BINTRAY_USER')
+    key = System.getenv('BINTRAY_API_KEY')
+
+    pkg {
+        repo = 'maven'
+        name = archivesBaseName
+        licenses = ['Apache-2.0']
+        vcsUrl = 'https://github.com/snowplow-referer-parser/java-referer-parser'
```

Comment:
I think we need the `.git` extension

Reply:
You are right (verified examples in the gradle-bintray-plugin). Fixed.